### DISCUSSION TITLE
Updating objects without getting the entry with a Get operation first

### DISCUSSION THREAD

**Discussion by RolandvonBorstel** [sadieblue/agilityblue.workspace-scripts#7] [Q&A] [answered]
I am wondering if it is possible to update one single custom field in a system object without retrieving the entry first using a Get operation. The idea is to reduce the number of function calls to speed up the execution of a script.

**Comment by bmward80** [comment, marked as answer]
Not at this time. We intend to introduce commands like you're describing that we call "patching" at some point, but for now, you'll need to retrieve the full object because the system requires it when using `Set` based commands for updating. If you really need this kind of functionality sooner rather than later, you'd need to set your trigger to call an outside service (like an Azure Function or AWS Lambda) that could take advantage of issuing PATCH commands directly via the API.

Brandon
